src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Footer } from "./Footer";
+
+export default {
+  title: "USWDS/Base/Footer",
+  component: Footer,
+  argTypes: {
+    emailAddress: {
+      description: "Email Address used at the bottom left for contact help",
+    },
+    address: {
+      description: "Address used at the bottom right of footer",
+    },
+    altFooter: {
+      description: "Determines whether to use the alternative footer",
+    },
+    navigationLinks: {
+      description:
+        "Array of navigation elements to render in the alternative footer",
+    },
```

Comment:
```suggestion
    navigationLinks: {
      description:
        "Array of navigation elements to render in the alternative footer",
        control: false,
    },
```
I think this is a good candidate for control to disable in Storybook.